frontend/src/components/worksheets/items/TableItem/TableItem.js

```diff
@@ -140,7 +140,21 @@ class TableItem extends React.Component<{
                     onMouseLeave={(e) => this.setState({ hovered: false })}
                     component='th'
                     key={index}
-                    style={index === 0 ? { paddingLeft: editPermission ? '30px' : '70px' } : {}}
+                    style={
+                        index === 0
+                            ? {
+                                  paddingLeft: editPermission ? '30px' : '70px',
+                                  position: 'sticky',
+                                  top: 0,
+                                  backgroundColor: '#F8F9FA',
+                                  zIndex: 1,
+                              }
+                            : {
+                                  position: 'sticky',
```

Comment:
Why are we making all the other headers (even those at indices > 0) sticky as well?

Reply:
I think the user requires to lock all the columns, such as uuid, name, epochs ...? So here if we only lock the header for whose index = 0, only the first column, i.e. the <uuid> column will be locked, and it will look like this:
<img width="940" alt="Screen Shot 2020-09-14 at 6 06 00 PM" src="https://user-images.githubusercontent.com/34461466/93153111-542d7480-f6b5-11ea-86b0-090dd6fc5435.png">. So that's why I make all the headers sticky.

Reply:
Ah, I see index is for columns, not rows.

Could we just set zIndex to 1 for all the columns so that we can remove the conditional to simplify the code?

Reply:
But besides the zIndex, we also need the conditional for deciding the `paddingLeft`? And it cannot be removed? (Cuz only the first column needs the left padding.)

Reply:
Got it, makes sense, so we can't remove the conditional.

Why do we need to conditionally set the zIndex though?

Reply:
I just set zIndex for all the columns. I previously conditionally set the Index because we only need to uplift the zIndex for the 1st column to make sure the 1st column covers the checkbox when scrolling. But I guess it would be good if we keep the zIndex for all the headers the same.